consensus/src/experimental/buffer_manager.rs

```diff
@@ -164,6 +167,60 @@ impl StateManager {
         }
     }
 
+    async fn process_ordered_blocks(
+        &mut self,
+        ordered_blocks: OrderedBlocks,
+    ) -> anyhow::Result<()> {
+        let OrderedBlocks {
+            blocks,
+            finality_proof,
```

Comment:
we should rename this to ordered_proof to avoid confusion

Reply:
resolved